lib/tapioca/generators/base.rb

```diff
@@ -0,0 +1,23 @@
+# typed: strict
+# frozen_string_literal: true
+
```

Comment:
Should we move the `require "thor" # TODO: Remove me when logging logic has been abstracted.` here?